netbeans.apache.org/src/content/about/history.asciidoc

```diff
@@ -0,0 +1,166 @@
+////
+     Licensed to the Apache Software Foundation (ASF) under one
+     or more contributor license agreements.  See the NOTICE file
+     distributed with this work for additional information
+     regarding copyright ownership.  The ASF licenses this file
+     to you under the Apache License, Version 2.0 (the
+     "License"); you may not use this file except in compliance
+     with the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing,
+     software distributed under the License is distributed on an
+     "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+     KIND, either express or implied.  See the License for the
+     specific language governing permissions and limitations
+     under the License.
+////
+= Apache NetBeans History
+:jbake-type: page
+:jbake-tags: about
+:jbake-status: published
+:keywords: Apache NetBeans History
+:description: Apache NetBeans History
+:toc: left
+:toc-title:
+:icons: font
+
+== Apache NetBeans History
+
+NetBeans started as a student project (originally called Xelfi) in
+Czechoslovakia, now the Czech Republic, in 1996. The goal was to write a
+Delphi-like Java IDE (Integrated Development Environment) in Java. Xelfi was
```

Comment:
Maybe link to https://en.wikipedia.org/wiki/Delphi_(IDE) for those that don't know what Delphi was.